src/Microsoft.Azure.WebJobs.Shared/Sanitizer.cs

```diff
@@ -18,7 +18,7 @@ internal static class Sanitizer
 
         // List of keywords that should not be replaced with [Hidden Credential]
```

Comment:
This is a shared file but there seems to be some difference in code between this file and the one in Host - [Sanitizer.cs](https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Sanitizer.cs). What's the guidance for this?

Reply:
Yes - the goal is to keep making the same identified improvements to each over time. The class comments on this Type indicate that "Note: This file is shared between the WebJobs SDK and Script repos. Update both if changes are needed."

Look at the file history of each - I think we revved them both recently.